stock_release_channel_geoengine/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,4 @@
+This module enhance release channels with the addition of
+a delivery zone that can be selected by the delivery manager through the UI.
+If a zone is specified, the release channel will exclusively select deliveries
+for partners who are localized within that zone.
```

Comment:
I would add the `ÌNSTALLATION.rst`file to explain that the module depends on the `base_geoengine` addon to compute whether a contact belongs to a geographical area. It therefore requires the use of the postgis extension module for postgresql...